Formula/fastnetmon.rb

```diff
@@ -0,0 +1,83 @@
+class Fastnetmon < Formula
+  desc "DDoS detection tool with sFlow, Netflow, IPFIX and port mirror support"
+  homepage "https://github.com/pavel-odintsov/fastnetmon/"
+  url "https://github.com/pavel-odintsov/fastnetmon/archive/refs/tags/v1.2.2.tar.gz"
+  sha256 "4de0fe9390673f7e2fc8f3f1e3696a1455ea659049430c4870fcf82600c2ea2d"
+  license "GPL-2.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "abseil"
+  depends_on "boost"
+  depends_on "capnp"
+  depends_on "grpc"
+  depends_on "hiredis"
+  depends_on "json-c"
+  depends_on "log4cpp"
+  depends_on macos: :big_sur # We need C++ 20 available for build which is available from Big Sur
+  depends_on "mongo-c-driver"
+  depends_on "openssl@1.1"
+
+  on_linux do
+    depends_on "gcc"
+    depends_on "libpcap"
+    depends_on "ncurses"
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    system "cmake", "-S", "src", "-B", "build",
+      "-DENABLE_CUSTOM_BOOST_BUILD=FALSE",
+      "-DDO_NOT_USE_SYSTEM_LIBRARIES_FOR_BUILD=FALSE",
+      "-DLINK_WITH_ABSL=TRUE",
+      "-DSET_ABSOLUTE_INSTALL_PATH=OFF",
+      "-DCMAKE_SYSTEM_INCLUDE_PATH=#{Formula["openssl@1.1"].include}",
+      "-DCMAKE_SYSTEM_LIBRARY_PATH=#{Formula["openssl@1.1"].lib}", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  service do
+    run [
+      opt_sbin/"fastnetmon",
+      "--configuration_file",
+      etc/"fastnetmon.conf",
+      "--log_to_console",
+      "--disable_pid_logic",
+    ]
+    keep_alive false
+    working_dir HOMEBREW_PREFIX
+    log_path var/"log/fastnetmon.log"
+    error_log_path var/"log/fastnetmon.log"
+  end
+
+  test do
+    cp etc/"fastnetmon.conf", testpath
+
+    inreplace testpath/"fastnetmon.conf", "/tmp/fastnetmon.dat", testpath/"fastnetmon.dat"
+
+    inreplace testpath/"fastnetmon.conf", "/tmp/fastnetmon_ipv6.dat", testpath/"fastnetmon_ipv6.dat"
```

Comment:
If `/tmp` is hardcoded in the `conf` file, I wonder if we should be pointing that to `var/"fastnetmon/run"` for more general use instead. Just thinking out loud -- we don't necessarily have to do this.

Reply:
We have plans to switch to API client and get rid of these files in coming releases.